gqueries/modules/multi_year_charts/primary_demand_chart/myc_primary_demand_of_geothermal_heat.gql

```diff
@@ -0,0 +1,3 @@
+- unit = PJ
+- query = 
+    Q(aggregated_total_primary_demand_of_geothermal_heat)
```

Comment:
For some reason we actually do not register geothermal heat as primary demand. This is a longstanding issue that should be picked up at some point: https://github.com/quintel/etmodel/issues/3641

Reply:
@kaskranenburgQ looking in the sankey_csv I notice that we should probably update `sankey_total_primary_demand_of_geothermal`. It uses outdated hardcoded geothermal nodes.

Reply:
Hmm what should be remove the primary demand queries? 
In the `sankey_total_primary_demand_of_geothermal `, we should indeed come up with a robust query at the moment the node `energy_heat_well_geothermal` is in there, while these have been replaced by deep, shallow, mt and ht wells

Reply:
What do you think of this :
`   DIVIDE(
      SUM(
        V(FILTER(G(heat_production),"input_of_geothermal > 0.0 "),"input_of_geothermal")
        ),
      BILLIONS
    )`

Reply:
`geothermal_input_conversion > 0.0" instead of "input_of_geothermal > 0.0". Other than that, all good!